elastic/integrations
# [Elasticsearch] Include `event.module` field for Elasticsearch Integration's File Datasets

- Enhancement

## What does this PR do?

Include `event.module: elasticsearch` for backward compatibility with dashboards, visualizations, alerts, etc. that were based on the Elasticsearch module for Filebeat. 

## Checklist


## Author's Checklist

- [X] Modified sample documents and fields

## Testing
- start stack `elastic-package stack up -v`
- start elasticsearch service `elastic-package service up -v`
- install elasticsearch integration and ingest logs. Logs are available under `/tmp/service_logs/<log-file>`
- ensure `event.module` is set to `elasticsearch` for log entries

## Related issues

- Closes #5703
